praas-app-ionic/src/styles/common.scss

```diff
@@ -18,6 +18,7 @@ ion-content {
   text-align: left;
 }
 
+.ion-no-padding,
```

Comment:
Isnt there already a class called ion-no-padding? Is this necessary?

Reply:
To not confuse, the code earlier added was to remove padding from the `IonItem`. ion-no-padding is already existing and neednt be added here